appveyor.yml

```diff
@@ -0,0 +1,53 @@
+version: "{branch}-ci-{build}"
+os:
+- Visual Studio 2013
```

Comment:
It must be another Appveyor bug. Boost 1.59 has the right MSVC14 version, the other do not.